src/components/DocExplorer.js

```diff
@@ -103,12 +114,25 @@ export class DocExplorer extends React.Component {
           field={typeOrField}
           onClickType={this._onClickTypeOrField}
         />;
+    } else if (this.state.searchValue) {
+      title = 'Search Results';
+      content = this.state.searchValue.length > 2 ?
+        <SearchDoc
+          searchValue={this.state.searchValue}
+          schema={schema}
+          onClickType={this._onClickTypeOrField}
+          onClickField={this._onClickTypeOrField}
+        /> :
+        <span className="doc-alert-text">
+          Minimum search length is 3 characters.
```

Comment:
I don't think we need to explicitly show this on the screen. Also, in a separate PR later we may want to remove this restriction so you could search for things like "id" but perhaps just change the search algorithm so it's not overwhelmed by < 3 ch searches.